appeals/web/src/server/appeals/appeal-documents/appeal-documents.validators.js

```diff
@@ -2,6 +2,43 @@ import { createValidator } from '@pins/express';
 import { body } from 'express-validator';
 import { createDateInputFieldsValidator } from '#lib/validators/date-input.validator.js';
 import { dateIsValid, dateIsTodayOrInThePast } from '#lib/dates.js';
+import { folderPathToFolderNameText } from '#appeals/appeal-documents/appeal-documents.mapper.js';
+
+export const validateDocumentNameBodyFormat = createValidator(
+	body()
+		.custom((bodyFields) => {
+			const items = Object.keys(bodyFields);
+			return items.includes('documentId') && items.includes('fileName');
+		})
+		.withMessage('There is a problem with the service')
+);
+
+export const validateDocumentName = createValidator(
+	body('fileName')
+		.trim()
+		.notEmpty()
+		.withMessage('Filename must be entered')
+		.bail()
+		.matches('^[a-zA-Z0-9_-]+.[a-zA-Z0-9_-]+$')
```

Comment:
It would be good to document what this regex does in a comment, just in case the error message content (which currently explains it well) needs to change

Reply:
Added comment as requested to explain the regex